src/overlays/actors/ovl_En_Mm2/z_en_mm2.c

```diff
@@ -43,33 +48,292 @@ static ColliderCylinderInit D_80AAFAC0 = {
     },
     { 18, 63, 0, { 0, 0, 0 } },
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Mm2/func_80AAEE50.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Mm2/func_80AAEF70.s")
+typedef struct {
+    /* 0x00 */ AnimationHeader* animation;
+    /* 0x04 */ f32 playSpeed;
+    /* 0x08 */ u8 mode;
+    /* 0x0C */ f32 morphFrames;
+} unknownStruct; // size = 0x10
```

Comment:
a struct for animation entries like this is pretty common. you can call it `Mm2AnimationEntry` or something along those lines